bus.css

```diff
@@ -0,0 +1,68 @@
+/*reset*/
+
+* {
+  width: 100%;
```

Comment:
Width: 100% in the * can do some unexpected things to your styling.  It will make normally inline elements behave as block elements (e.g. span, img, etc).